engine/src/main/java/org/destinationsol/game/SolCamera.java

```diff
@@ -73,20 +76,20 @@ public void update(SolGame game) {
                 myPos.set(trans.getPosition());
             }
         } else {
-            Vector2 heroPos = hero.getHull().getBody().getWorldCenter();
+            Vector2 heroPos = hero.orElseThrow(SolNullOptionalException::new).getHull().getBody().getWorldCenter();
```

Comment:
The `orElseThrow` clause will never be called, since `hero.isPresent()` is true. Might make sense to unwrap the value inside the optional into a local variable.